content/2017-07-19_djangozeny.md

```diff
@@ -8,53 +8,41 @@ About: Nikdo. Nic. Nikam.
 
 ![Atmosféra na workshopu]({filename}/images/IMG_0518.jpg)
 
 Musím říct, že Zuzka s Verčou odhalily tajemství spokojených účastnic a strategicky zahájily workshop snídaní. Mňam!
-Po krátkém představení koučů a programu se už každá účastnice vrhla na tutoriál. Princip miniskupinek s koučem a postupování
-každé vlastním tempem mi vyhovoval na 1000%. Stačilo se zoufale podívat koučovým směrem nebo zmínit, že „vono to nefunguje“
+Po krátkém představení koučů a programu se už každá účastnice vrhla na tutoriál. Princip miniskupinek s koučem a postupování vlastním tempem mi vyhovoval na 1000 %. Stačilo se zoufale podívat koučovým směrem nebo zmínit, že „vono to nefunguje“
 a během chvilky „to“ už zase fungovalo. Nevím jak u ostatních stolů, ale náš Pepa se občas docela nudil, což nepřipisuju
-na vrub naší skupince, ale fakt skvěle napsanému tutoriálu, který obsahoval i kdy si dojít pro kafe nebo občerstvení,
+na vrub naší skupince, ale fakt skvěle napsanému tutoriálu, který nám i připomínal, kdy si dojít pro kafe nebo občerstvení,
 kterého bylo po celý den víc než dost.
 
 ![Oběd na terase]({filename}/images/IMG_0343.jpg)
 
 Ani jsem se nenadála a už byl čas na oběd. Opět musím vyseknout poklonu objednanému cateringu, protože jídlo bylo výborné,
-pestré a bylo ho dost. :) Po jídle jsme byly vyhnány na velkou terasu se zahrádkou a krásným výhledem na společné focení,
-aby jste pak všichni viděli jak jsme se tam měli dobře. Posoudit můžete [„zde“](https://www.dropbox.com/sh/6s2kyttxdq4g9zr/AABOSpfBOIzIYhR-ktJ0eSLea?dl=0). Musím říct, že mě ten worshop bavil na tolik,
-že jsem si i oběd vzala k počítači a pro kafe běhala minimálně, a to bylo fakt dobrý! :) Druhá půlka workshopu byla
-trochu zajímavější, jelikož po <var>x</var> hodinách všemožného nastavování jsme se dostali i k programování. Vševědoucí Pepa se trpělivě skláněl nad našima notebookama a hledal zapomenuté závorky, překlepy, řešil nevysvětlitelné errory a odpovídal na všetečné otázky typu „A proč jako? / A proč jako ne takhle?“
+pestré a bylo ho dost. Po jídle jsme byly vyhnány na velkou terasu se zahrádkou a krásným výhledem na společné focení,
+abyste pak všichni viděli, jak jsme se tam měli dobře. Posoudit můžete [„zde“](https://www.dropbox.com/sh/6s2kyttxdq4g9zr/AABOSpfBOIzIYhR-ktJ0eSLea?dl=0). Musím říct, že mě workshop bavil natolik,
+že jsem si i oběd vzala k počítači a pro kafe běhala minimálně, a přitom bylo fakt dobrý! :) Druhá půlka workshopu byla
```

Comment:
Prosím nechat původní znění :)

.... a pro kafe běhala minimálně, a to bylo fakt dobrý! :) Druhá půlka....